core/assets/bundles/bundle_ru.properties

```
@@ -1002,8 +1002,8 @@ setting.difficulty.hard = Сложная
 setting.difficulty.insane = Безумная
 setting.difficulty.name = Сложность:
 setting.screenshake.name = Тряска экрана
-setting.bloomintensity.name = Bloom Intensity
-setting.bloomblur.name = Bloom Blur
+setting.bloomintensity.name = Интенсивность цвета
```

Разве "bloom" - это не свечение?

```suggestion
setting.bloomintensity.name = Интенсивность свечения
```

---

Сам уже запутался хд